File: Miguel Franco, Oswaldo Romero/Tareas/Separar Entero y Float/Separar Entero y Float.py
# Solicitar al usuario que ingrese un número
numero = float(input("Ingresa un número: "))

# Separar la parte entera de la decimal
parte_entera = int(numero)
parte_decimal =round(numero - parte_entera, 4)
# Imprimir los resultados
print("Parte entera:", parte_entera)
print("Parte decimal:", parte_decimal)

File: Miguel Franco, Oswaldo Romero/Tareas/Separar Entero y Float/Separar Entero y Float.cpp
#include <stdio.h>

main(){
int a;
float b,c;

printf("Introduzca un numero: ");
scanf("%f",&b);

a=b;
c=b-a;

printf("La parte entera es %d\n",a);
printf("La parte decimal es %.4f\n",c);

return 0;

}

